docs/RECIPES.md

```diff
@@ -137,3 +137,33 @@ EOF
 ```shell
 oc apply -f preflight.yaml
 ```
+
+## Container Policy
+These examples are shown using the Container policy against a container image
+(e.g. `preflight check container <image>`). Container policy only runs as a binary on your workstation. Check the latest
+[release](https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases) for the binary that matches your operating system.
+
+You will also need:
+- Your container image published to a container registry
+- A Certification Project ID of the project that was set up in Red Hat Partner Connect
+- A Partner Connect API Key
+
+### Testing a Container
+Running container policy checks against a container iteratively until all tests pass.
+
+```bash
+preflight check container registry.example.org/your-namespace/your-image:sometag
```

Comment:
If you are going to multiline these, use a \ at the end of all lines but the last.

Reply:
good catch. ty!